### ISSUE TITLE
ERROR: Unable to find the Nextcloud container - due to tolower()

### ISSUE THREAD

**Issue by WGandy** [rcdailey/nextcloud-cronjob#12]
tldr: Pay attention to uppercase letters in your project folder name. 

I think this is something to either program around or at least document in your (rather nice) documentation. 

It seems that docker (or docker compose) removes all case from the folder name (project name). Since my folder had a couple capital letters, and that is how I typed it into the environment variable, your cron service couldn't find it. 

Once I understood, it was fairly simple to rename:  
    NEXTCLOUD_PROJECT_NAME=all_lower_case

While troubleshooting this I initially thought that there was an issue because of the way compose numbers the container names (nextcloud_nextcloud_1) but you seem to have figured out how to work around that.  

Thanks,

**Comment by rcdailey** [comment]
Is this related to #10 maybe? Do you see the name using hypens (`-`) instead of underscores (`_`)?

**Comment by WGandy** [comment]
I don't believe this has anything to do with V2 of Docker-Compose. I was using version 1.29.2 when I encountered this. 

My entire problem appeared to hinge on the upper case letters that I used for the NEXTCLOUD_PROJECT_NAME= variable. Once I lowered all of the characters in the variable (leaving my folder name as it was) it ran without error. 

For example: If my project folder is called "CaMeLcAsE" then docker-compose will use "camelcase" as the prefix for the container name (and apparently also for a separate searchable project variable).

**Comment by rcdailey** [comment]
I never toyed around with uppercase letters in directory names; so I'm not sure how Docker Compose actually worked in that case. I'm glad you were able to figure it out. I'll go ahead and close this issue. Thank you for the information!
